app/controllers/githook_controller.rb

```diff
@@ -1,20 +1,17 @@
 class GithookController < ApplicationController
-  before_action :login_required, :only => [:dev, :master]
-  before_action :authenticate, :only => [:dev, :master]
+  before_action :login_required, only: ['dev', 'master']
+  before_action :authenticate, only: ['dev', 'master']
+  before_action :verify_request_is_from_github, only: 'index'
 
   def index
     ref = params[:ref]
     success = false
-    if ref == "refs/heads/master"
+    if ref == 'refs/heads/master'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.